news/2022-07-31-the-sound-design-of-lazer.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: "The sound design of lazer"
+date: 2022-07-31 00:00:00 +0000
+---
+
+Learn about the inner workings behind the future sound of osu!
+
+![](/wiki/shared/news/2022-07-31-lazer-sound-design/banner.png)
+
+Sound design relates to how any particular sound can make its listener *feel*. The sound itself can be anything from the pop and fizz when opening a can of Coca-Cola, to the continuous rumble of a car engine. In video games, sounds need to match the things players interact with, so for osu! the question is: "what sounds should things like a circle, button, or a toggle have?"
+
+osu!'s new client intends to answer that question with lots of brand new changes, including brand new sound effects to suit the brand new UI. If you are wondering where they come from or how they are made, read along for an interview with [nekodex](https://osu.ppy.sh/users/102), the person behind the sound design of osu!(lazer).
+
+Jamie "nekodex" Taylor is a full-stack engineer born in Australia who joined the osu! team as a web developer in 2015 after peppy asked if he was interested in helping rebuild the osu! website. Jamie has been with the team ever since, working on various osu!-related things, such as the review system for beatmap modding, the chat system etc., but today he describes himself as a hybrid audio designer/developer and isn't too involved in the web side of things anymore. Today we'll take a look at the origins of nekodex, how he developed as an artist and the work he's done for the audio side of osu!.
+
+nekodex as an alias came to be in 2012, as Jamie had a musical background while growing up, played with early music software, learned from YouTube tutorials and eventually started messing around in DAWs (digital audio workstations) until music came out. As Jamie was getting serious with music production, peppy was preparing osu!stream (an iOS spinoff of osu! that was created in 2010) for the public view, which required original music for the game.
+
+Since peppy and Jamie are real life friends and peppy knew about his music, he got the chance to create some themes and tracks for osu!stream, and later on got some of his own songs mapped for it as well. He then later went on to create several official songs throughout the years that we all know and love, such as [welcome to osu!](https://soundcloud.com/nekodex/welcome-to-osu), [circles!](https://soundcloud.com/nekodex/circles) and [aureole](https://soundcloud.com/nekodex/aureole).
+
+When peppy decided to start working on a new client from the ground up in 2016 (after trying to make lazer within the stable client and figuring out such a thing would not be feasible), he wanted to not only make the game look good, but also *feel* good. That task required an audio designer, and with nekodex already having done several amazing works for osu!, peppy approached him.
```

Comment:
Yep, as mentioned. I approached nekodex in 2021 because the sound design wasn't up-to-bar with the visual experience, and wanted someone to take this on full time until there was parity there.

Reply:
Thanks both, I have reworded the article to convey this detail properly